cranelift/isle/isle/tests/run_tests.rs

```diff
@@ -50,5 +57,18 @@ pub fn run_link(isle_filename: &str) {
     assert!(rustc.wait().unwrap().success());
 }
 
+pub fn run_envs_for_analysis(filename: &str) {
+    match envs_for_analysis(filename) {
+        // Alexa TODO: is there a more general check that the term was not expanded?
```

Comment:
Also hoping for feedback here: is there a better way to test that the extractor has not been expanded, other than searching for a specific term name?